atomium/templates/pager_last/pager_last.component.inc

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @file
+ * Contains component file.
+ */
+
+/**
+ * Implements hook_atomium_theme_hook().
+ */
+function atomium_atomium_theme_pager_last(array $existing, $type, $theme, $path) {
+  return array(
+    'pager_last' => array(
+      'template' => 'pager-last',
+      'variables' => array(
+        'text' => NULL,
+        'element' => 0,
+        'interval' => 1,
+        'parameters' => array(),
+      ),
+    ),
+  );
+}
+
+/**
+ * Implements hook_atomium_definition_hook().
+ */
+function atomium_atomium_definition_pager_last() {
+  return array(
+    'label' => 'Pager last',
+    'description' => 'Pager last description',
+  );
+}
+
+/**
+ * Implements hook_preprocess_hook().
+ */
+function atomium_preprocess_pager_last(array &$variables, $hook) {
+  $text = $variables['text'];
+  $element = $variables['element'];
+  $parameters = $variables['parameters'];
+  global $pager_page_array, $pager_total;
+  $render = NULL;
+
+  $pager_last = NULL;
+
+  // If we are anywhere but the last page.
```

Comment:
What does mean the comment?

Reply:
If we are on any element except the last page element of the pager.